lib/createInlinePluginCreator.js

```diff
@@ -226,6 +229,14 @@ function createInlinePluginCreator(packages, multiContext, synchronizer, flags)
 			getLucky("_readyForTagging", pkg);
 			await waitFor("_readyForTagging", pkg);
 
+			getPkgRootOptions(context).forEach((pkgRoot) => {
+				const manifestPath = cleanPath(`${pkgRoot}/package.json`, context.cwd);
```

Comment:
If pkgRoot equals to `.`, shouldt we just skip this?

Reply:
And we also need a test for this feature too.

Reply:
What do you mean by `.` ?
You mean the current workspace path or like literally the point character ?
Is it to avoid doing it twice with the line [#240](https://github.com/dhoulb/multi-semantic-release/blob/171a32e4c5d05cb5fa0900164f0bd6734ee311ee/lib/createInlinePluginCreator.js#L240) ?

Reply:
Agreed for the tests.
_(will do once we are ok on the implem 😉)_